Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM busybox
+
+# The commit sha we wish to ship to prod
+ENV SHA master
```

Comment:
So I did this because we may not want to deploy master every time; i.e. we merged a few things but aren't ready to push them.